tikv/kv.go

```diff
@@ -172,8 +171,28 @@ func (s *KVStore) CheckVisibility(startTime uint64) error {
 	return nil
 }
 
+// Option is the option for pool.
+type Option func(*KVStore)
+
+// WithPool set the pool
+func WithPool(gp Pool) Option {
+	return func(o *KVStore) {
+		o.gP = gp
+	}
+}
+
+// loadOption load KVStore option into KVStore.
+func loadOption(store *KVStore, opt ...Option) {
+	if len(opt) == 0 {
+		store.gP = NewSpool(128, 10*time.Second)
```

Comment:
better to put it to when initializing KVStore (Line 213)

Reply:
Done